index.js

```diff
@@ -15,15 +15,35 @@ module.exports = function recommendations (options, overrides) {
     variationId: options.meta.variationId
   }, overrides)
 
+  function coerceToNumber (n) {
+    if (typeof n === 'number') {
+      return n
+    }
+    if (typeof n === 'string' && n.length > 0) {
+      n = n
+        .replace(/[^0-9,.]/g, '')
+        .replace(/,(?=\d{1,2}$)/, '.')
+        .replace(/(\s|,)/g, '')
+        .replace(/\.(?=[0-9]{3}(\.|,))/g, '')
```

Comment:
Does this have to be different from `uv-maps/lib/helpers/coerce-to-number.js`? I wonder if we have this somewhere in a library that may be shared between `uv-maps` and this module.

Reply:
Just checked `master` [here](https://github.com/qubitdigital/uv-maps/blob/master/lib/helpers/coerce-to-number.js) and the regexes are the same. No shared library, but the functions are the same.